src/cuga/backend/server/main.py

```
@@ -1629,6 +1629,95 @@ async def event_stream(
+
+        async def run(self, message, context_id=None):
+            try:
+                supervisor = await self._ensure_supervisor()
+                result = await supervisor.invoke(message, thread_id=context_id)
+                answer = getattr(result, "answer", None) or str(result)
+                error = getattr(result, "error", None)
+                if error:
+                    yield _A2AStreamEvent("error", {"text": f"Supervisor error: {error}"}, final=True)
+                    return
+                yield _A2AStreamEvent("final_answer", {"text": answer}, final=True)
+            except Exception as exc:  # surface failures across the wire
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)